OrbitGl/CaptureWindow.cpp

```diff
@@ -629,9 +629,16 @@ void CaptureWindow::ToggleDrawHelp() {
   NeedsRedraw();
 }
 
-Batcher& CaptureWindow::GetBatcherById(uint32_t batcher_id) {
-  return batcher_id == PickingID::TIME_GRAPH ? time_graph_.GetBatcher()
-                                             : ui_batcher_;
+Batcher& CaptureWindow::GetBatcherById(BatcherId batcher_id) {
+  switch (batcher_id) {
+    case BatcherId::kTimeGraph:
+      return time_graph_.GetBatcher();
+    case BatcherId::kUi:
+      return ui_batcher_;
+    default:
+      CHECK(false);
```

Comment:
Use `UNREACHABLE` or `FATAL` instead of `CHECK(false)`.

Reply:
Done